src/app/work-item/work-item-detail/markdown-control/markdown-control.component.html

```diff
@@ -54,6 +54,7 @@
   <div class="col-md-12 col-sm-12 col-xs-12 detail-text-action">
       <span class="col-md-4">
         <small *ngIf="loggedIn && tabBarVisible">
+          <img src="../../../../assets/img/markdown-solid.png" class="markdown-mark" />
           <b>Markdown Supported</b>
```

Comment:
Here's the steps that the CI pipeline does, this should recreate the error - can you check to see if the steps I'm doing are correct please?
```
git clone git@github.com:fabric8io/fabric8-planner.git
cd fabric8-planner
git fetch origin pull/1626/head:PR1626
git checkout PR1626
npm install
npm run build:prod
git clone git@github.com:fabric8io/fabric8-ui.git
cd fabric8-ui
npm install
npm install --save  ../dist
npm run build:prod
```